src/partials/footer.html

```diff
@@ -0,0 +1,29 @@
+<footer class="footer">
+  <div class="footer__container">
+    <div class="footer__logo">
+      <svg class="footer__icon">
+        <use href="./images/sprite.svg#icon-logo"></use>
+      </svg>
+      <div class="footer__tablet">
+        <div class="footer__logo--tel">
+          <svg class="footer__icon--orange" width="15px" height="12px">
```

Comment:
Modyfikator w BEM nie powinien być stosowany bez klasy głównej. Jeżeli użyty jest footer__icon--orange to powinien być również użyty footer__icon. Sugeruję zrobienie tutaj osobnej klasy np. footer__phone-icon i nadanie jej koloru oraz wysokości i szerokości, a usunięcie wysokości i szerokości z pliku HTML.